### ISSUE TITLE
WiFi Performance really bad

### ISSUE THREAD

**Issue by Thashiznets** [Ansuel/tch-nginx-gui#797]
Device Model: MediaAccess TG789MYRvac v2 HP
Gui Version: 9.4.22

Description of problem: WiFi has become very bad lately, on 5GHz with good signal strength normally I get about 45Mbps/12Mbps with my ISP but since last few updates the WiFi performance is so bad it is sometimes ok but if I test again straight after the latency goes up really high about 200ms (normally it is about 13ms) and I get 2Mbps/3Mbps type speeds very bad.

How to reproduce (if possible):

Just using WiFi 5Ghz on Channel 44.

**Comment by kevdagoat** [comment]
Try a different channel as the one you are on may be congested or in use

**Comment by Thashiznets** [comment]
> Try a different channel as the one you are on may be congested or in use

I scanned my wifi and there are no channels in that lower range in use around my, they are up in the 100's
[DebugHelper2019-09-02-1649.tar.gz](https://github.com/Ansuel/tch-nginx-gui/files/3564803/DebugHelper2019-09-02-1649.tar.gz)

^ I have attached the debug log, also it looks like it can't run UPNP according to error log?

**Comment by kevdagoat** [comment]
UPnP shouldn’t be a problem. Try running an IPerf test to determine if it is your internet or your wifi

**Comment by Thashiznets** [comment]
> UPnP shouldn’t be a problem. Try running an IPerf test to determine if it is your internet or your wifi

What I did is I used a computer connected via Ethernet and it didn't have issue but MacBook over 5GHz does. Interestingly I did change wifi channel to 153 and it is running a lot better. Unsure why as wifi scanner is not showing competition in the lower channels and even I have a competitor on channel 161 yet this channel is performing better. That is strange?

Did wifi driver get updated or something?
